src/OceanSeaIceModels/InterfaceComputations/assemble_net_fluxes.jl

```diff
@@ -121,22 +127,25 @@ end
     ρₒ⁻¹ = 1 / ocean_properties.reference_density
     cₒ   = ocean_properties.heat_capacity
 
-    τxao = ℑxᶠᵃᵃ(i, j, 1, grid, τᶜᶜᶜ, ρₒ⁻¹, ℵ, ρτx)
-    τyao = ℑyᵃᶠᵃ(i, j, 1, grid, τᶜᶜᶜ, ρₒ⁻¹, ℵ, ρτy)
-    Jᵀao = ΣQao  * ρₒ⁻¹ / cₒ
-    Jˢao = - Sₒ * ΣFao
-
-    ρₒ⁻¹ = 1 / ocean_properties.reference_density
-    cₒ   = ocean_properties.heat_capacity
-
     @inbounds begin
         ℵᵢ   = ℵ[i, j, 1]
         Qio  = sea_ice_ocean_fluxes.interface_heat[i, j, 1]
-        Jˢio = sea_ice_ocean_fluxes.salt[i, j, 1]
+
+        Jᵀao = ΣQao  * ρₒ⁻¹ / cₒ
+        Jˢao = - Sₒ * ΣFao
         Jᵀio = Qio * ρₒ⁻¹ / cₒ
+        Jˢio = sea_ice_ocean_fluxes.salt[i, j, 1]
+
+        τxao = ℑxᶠᵃᵃ(i, j, 1, grid, τᶜᶜᶜ, ρₒ⁻¹, ℵ, ρτxao)
+        τyao = ℑyᵃᶠᵃ(i, j, 1, grid, τᶜᶜᶜ, ρₒ⁻¹, ℵ, ρτyao)
+        τxio = ρτxio[i, j, 1] * ρₒ⁻¹ * ℑxᶠᵃᵃ(i, j, 1, grid, ℵ)
+        τyio = ρτyio[i, j, 1] * ρₒ⁻¹ * ℑyᵃᶠᵃ(i, j, 1, grid, ℵ)
+
+        # Stresses
+        τx[i, j, 1] = τxao + τxio
+        τy[i, j, 1] = τyao + τyio
 
-        τx[i, j, 1] = τxao
-        τy[i, j, 1] = τyao
+        # Tracer fluxes
         Jᵀ[i, j, 1] = (1 - ℵᵢ) * Jᵀao + Jᵀio
         Jˢ[i, j, 1] = (1 - ℵᵢ) * Jˢao + Jˢio
```

Comment:
Just double checking that these two lines are correct.
Intuitively I'd expect them to be something like:

```julia
J[i, j, 1] = (1 - ℵᵢ) * Jao + ℵᵢ * Jio
```

so that when `ℵᵢ = 0` we have `Jao` and when `ℵᵢ = 1` we have `Jio`. But perhaps `ℵᵢ` is incorporated in the `Jio`?

Reply:
I think you are right. `ℵᵢ` is incorporated in `Jio`, but it is probably better not to do it to make the code more clear. Let's change it in another PR.